app/assets/stylesheets/application.scss

```diff
@@ -12,6 +12,97 @@
 
 .wrapper {
   @extend %site-width-container;
+  float: left;
+  clear: left;
+  margin-top: 60px;
+}
+
+.control-indicator {
+  position: relative;
+  bottom: 30px;
+  width: 39px;
+  height: 39px;
+  background: $white;
+  border-radius: 100%;
+  border: 2px solid $black;
+  cursor: pointer;
+}
+
+.control-indicator:after {
+  content: "";
+  position: relative;
+  color: $black;
+  border: 5px solid $black;
+  left: 6px;
+  top: 6px;
+  width: 23px;
+  height: 23px;
+  border-radius: 50%;
+  background: $black;
+  display: none;
+}
+
+.control-indicator:focus,
+.form-control:focus {
+  outline-offset: 0;
+  outline: 3px solid $yellow;
+  border: 2px solid $black;
+}
+
+.doc-link {
+  font-size: 14px;
+}
+
+.down-box,
+.down-box a:link,
+.down-box a:visited {
+  color: $red;
+}
+
+.form-control {
+  border: 2px solid $black;
+  border-radius: 0;
+  font-size: 19px;
+  line-height: 1.31579;
+}
+
+.form-label {
+  @include heading-24;
+  display: block;
+  float: left;
+  clear: left;
+}
+
+.form-label input:checked ~ .control-indicator:after {
+  display: block;
+}
+
+.head-section {
+  background-color: $grey-3;
+  @extend %grid-row;
+
+  .results-table {
+    @include grid-column( 3 / 3 );
+  }
+
+}
+
+.head-info {
+  @include copy-14;
+  color: $grey-1;
+  position: relative;
+  bottom: 20px;
+}
+
+.head-info a:link {
+  @include copy-14;
+}
+
+.head-info-2 {
```

Comment:
yeah it covers the description, I separated it out to make the view a bit clearer